index.html

```diff
@@ -108,63 +128,74 @@ <h4>Profesional Art Printing Data</h4>
-          <h5>Languages</h5>
+        <div class="aboutme-Languages">
+          <div class="all">
+            <img src="images/Diagonal.png" alt="square-image">
+            <h5>Languages</h5>
+          </div>
+            <ul>
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>Css</li>
+            </ul>
         </div>
-          <ul>
-            <li>Javascript</li>
-            <li>Ruby</li>
-            <li>Html</li>
-            <li>Css</li>
-          </ul>
-        <div class="all">
-          <img src="images/Rectangle.png" alt="square-image">
-          <h5>Frameworks</h5>
+        <div class="aboutme-Languages">
+          <div class="all">
+            <img src="images/Rectangle.png" alt="square-image">
+            <h5>Frameworks</h5>
+          </div>
+            <ul>
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Elenium</li>
+            </ul>
         </div>
-          <ul>
-            <li>Bootstrap</li>
-            <li>Ruby on Rails</li>
-            <li>RSPec</li>
-            <li>CapyBara</li>
-            <li>Elenium</li>
-          </ul>
-        <div class="all">
-          <img src="images/Circle.png" alt="circle-image">
-          <h5>Skills</h5>
+        <div class="aboutme-Languages">
+          <div class="all">
+            <img src="images/Circle.png" alt="circle-image">
+            <h5>Skills</h5>
+          </div>
+            <ul>
+              <li>Codekit</li>
+              <li>Guthub</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
         </div>
-          <ul>
-            <li>Codekit</li>
-            <li>Guthub</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
-          </ul>
       </div>
     </section>
     <section class="contact-form">
-      <p>
-        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
-      </p>
-      <form action="https://formspree.io/f/mnqwdall" method="post">
-        <input class="contact-name" type="text" name="full name" id="signup" maxlength="30" placeholder="Full name" required>
-        <hr class="form-line">
-        <input class="contact-email" type="email" name="email" id="fillup" placeholder="Email address" required>
-        <hr class="form-line">
-        <textarea name="msg" id="textup" cols="25" rows="4" maxlength="500" placeholder="Enter text here" required></textarea>
-        <input type="submit" class="contact-button" value="Get in touch"/>
-      </form>
+      <div class="contact-section_form">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </p>
+        <form action="https://formspree.io/f/mnqwdall" method="post">
```

Comment:
- [ ] Great job on creating the formsrpee on this form, however, it is not yet functional, I tried to get in touch either on mobile or desktop, but still the same, kindly fix this issue.
- [ ] Talking about the design, this section needs some improvement too, see this image below and please adjust the alignment shown.
<img width="282" alt="havabt" src="https://user-images.githubusercontent.com/79058364/154363191-4b98b1aa-81f8-41a4-8982-120332758942.png">